Services/Cron/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron
+
```

Comment:
Maybe we should add some RFC introduction here.

The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.